schematic/__init__.py

```diff
@@ -139,17 +183,6 @@ def request_hook(span: Span, environ: Dict) -> None:
     """
     if not span or not span.is_recording():
         return
-    try:
-        if auth_header := environ.get("HTTP_AUTHORIZATION", None):
-            split_headers = auth_header.split(" ")
-            if len(split_headers) > 1:
-                token = auth_header.split(" ")[1]
-                user_info = info_from_bearer_auth(token)
-                if user_info:
-                    span.set_attribute("user.id", user_info.get("sub"))
-    except Exception:
```

Comment:
I know you're still working on this, but just a reminder, should this be added back in?

Reply:
hmm I don't think we need this part any more because we are not using the `info_from_bearer_auth` function to debug the token. What's the purpose of keeping it?

Reply:
This portion does not need to be kept